drush/sites/self.site.yml

```diff
@@ -11,4 +11,15 @@
 #  host: stage.domain.com
 #  user: www-admin
 #  root: /path/to/drupal
-#  uri: http://stage.example.com
\ No newline at end of file
+#  uri: http://stage.example.com
+#
+# For Silta environments, you can use this wildcard alias.
+# Replace [repo-name] with the name of your repository, and leave out the square brackets.
+# Usage: drush @branch-name status
+#'*':
+#  uri: https://${env-name}.[repo-name].silta.wdr.io
```

Comment:
Actually the `[repo-name]` does not apply here if silta.yml `projectName` defines the domain. Would it be too confusing here to example as
```
uri: https://${env-name}.[repo-name OR projectName from silta.yml].silta.wdr.io
```